types/update-notifier/index.d.ts

```diff
@@ -7,6 +7,7 @@
 //                 Piotr Błażejewicz <https://github.com/peterblazejewicz>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
+import boxen = require('boxen');
```

Comment:
this requires `package.json` to be added with the PR.
Adding dependency is OK, as the `update-notifier` depends on `boxen`

Reply:
Awesome, thanks, I'll look into it! Apologies for the dumb mistakes, DefinitelyTyped noob here!

Out of curiosity, do you know why `configstore` below didn't need a `package.json`?

Reply:
Oh I think that'd be it:

> A package.json may be included to specify dependencies that are not other `@types` packages.

From https://github.com/DefinitelyTyped/DefinitelyTyped#i-notice-some-packages-having-a-packagejson-here

`configstore` is a `@types` package: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/configstore/index.d.ts

Reply:
Done at https://github.com/microsoft/DefinitelyTyped-tools/pull/74